src/stores/AppStore/AppStore.ts

```diff
@@ -1310,6 +1310,60 @@ export class AppStore {
         }
     }
 
+    /**
+     * Deletes all regions including annotations.
+     */
+    @action deleteAllRegions = () => {
+        this.activeFrame.regionSet.regionMap.forEach(x => {
+            if (x.regionId !== CURSOR_REGION_ID) {
+                this.deleteRegion(x);
+            }
+        });
+    };
+
+    /**
+     * Deletes all annotations.
+     */
+    @action deleteAllAnnotations = () => {
+        this.activeFrame.regionSet.regionMap.forEach(x => {
+            if (
+                x.regionId !== CURSOR_REGION_ID &&
+                (x.regionType === CARTA.RegionType.ANNCOMPASS ||
```

Comment:
We could use `RegionStore.isAnnotation`.

Reply:
thanks.